### DISCUSSION TITLE
Set up Home directory for specific SFTP user

### DISCUSSION THREAD

**Discussion by paratz** [AzuraCast/AzuraCast#4221] [Q&A] [answered]
Hello Everyone, first of all thanks to the AzuraCast team for all your effort on this project!

Is there any way to setup a home directory for a SFTP user other than the media folder? We want to delegate the media management of a specific show to somebody, but we don't want him to see/change all the media files in the station. The folder for this show is a directory within the media folder of the station.

Thanks for your comments,

**Comment by BusterNeece** [comment, marked as answer]
@paratz That's not currently possible, but it is technically possible using the software stack we have if we were to add the configuration option to the SFTP database table; this would be a good candidate for a [feature request](https://features.azuracast.com/)

**Reply by paratz** [reply]
Thanks for your reply,
